src/app/source-file-tabs/source-file-tabs.service.ts

```diff
@@ -131,19 +128,9 @@ export class SourceFileTabsService {
     const id = ga4ghWorkflowIdPrefix + fileId;
     return this.fileService.getDownloadFilePath(id, versionName, type, relativePath);
   }
-  getPrimaryPath(
-    path: string,
-    versionName: string,
-    descriptorType: ToolDescriptor.TypeEnum,
-    descriptorPath: string,
-    entryType: EntryType
-  ): string[] {
+  getPrimaryPaths(descriptorType: ToolDescriptor.TypeEnum, descriptorPath: string): string[] {
     const primaryPath = [];
-    primaryPath.push(this.infoTabService.getPrimaryPath(path, versionName, descriptorType, descriptorPath, entryType));
-    if (descriptorType === 'NFL') {
-      primaryPath.push('main.nf');
-      primaryPath.push('/main.nf');
-    }
+    primaryPath.push(descriptorPath);
     return primaryPath;
```

Comment:
I was going to remove the method but I was thinking maybe to leave it since we are going to look into how we can handle Nextflow and other types better - possibly having more than one primary path. Should I add a comment or TODO for the method? Or do you think deleting it would be best?